### PR TITLE
Pivotal #ID 180345449: handle empty title submission email notification

### PR DESCRIPTION
**Pull request by Juan-EBI**
https://www.pivotaltracker.com/story/show/180345449

### CODE REVIEW

File: commons/commons-model-extended/src/test/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensionsTest.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        val submissionTitle = testSubmission(subTitle = "submission title", secTitle = null)
+        val submissionNoTitleSecTitle = testSubmission(subTitle = null, secTitle = "section title")
+        val submissionNoTitleNoSecTitle = testSubmission(subTitle = null, secTitle = null)
+
+        assertThat(submissionTitle.description).isEqualTo("submission title")
+        assertThat(submissionNoTitleSecTitle.description).isEqualTo("section title")
+        assertThat(submissionNoTitleNoSecTitle.description).isNull()
+    }
+
+    private fun testSubmission(subTitle: String? = null, secTitle: String? = null): ExtSubmission {
```
Comment (jhoanmanuelms) [inline commons/commons-model-extended/src/test/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensionsTest.kt:20]: One-liner?

File: commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensions.kt [hunk 2]
```diff
@@ -16,3 +16,6 @@ val ExtSubmission.isCollection
 
 val ExtSubmission.filesPath
     get(): String = "$relPath/Files"
+
+val ExtSubmission.description
```
Comment (jhoanmanuelms) [inline commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensions.kt:20]: I think it's confusing that the description actually returns the title. We should call it something like `submissionTitle` or something like that.

**Review by jhoanmanuelms — APPROVED** [review]
LGTM. Please consider my comment before merging.

File: commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensions.kt (hunk 2 above)
Comment (Juan-EBI) [reply commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensions.kt:20]: I had issues naming this as sometimes is the section title and submissionTitle is kind of redudant as it bellows to submission object and is confusing with the title attribute itself.

**Pull request merged**